[2021-02-18 22:48:04,281] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:47:00+00:00 [queued]>
[2021-02-18 22:48:04,287] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:47:00+00:00 [queued]>
[2021-02-18 22:48:04,287] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:48:04,287] {taskinstance.py:1018} INFO - Starting attempt 1 of 2
[2021-02-18 22:48:04,287] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:48:04,297] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): get_tweets_by_user_list> on 2021-02-18T22:47:00+00:00
[2021-02-18 22:48:04,299] {standard_task_runner.py:51} INFO - Started process 19304 to run task
[2021-02-18 22:48:04,304] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'tweets_by_user', 'get_tweets_by_user_list', '2021-02-18T22:47:00+00:00', '--job-id', '1156', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tweets_by_user.py', '--cfg-path', '/tmp/tmpkh63ff6n']
[2021-02-18 22:48:04,307] {standard_task_runner.py:76} INFO - Job 1156: Subtask get_tweets_by_user_list
[2021-02-18 22:48:04,335] {logging_mixin.py:103} INFO - Running <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:47:00+00:00 [running]> on host ip-172-31-3-59.ec2.internal
[2021-02-18 22:48:04,360] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tweets_by_user
AIRFLOW_CTX_TASK_ID=get_tweets_by_user_list
AIRFLOW_CTX_EXECUTION_DATE=2021-02-18T22:47:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-18T22:47:00+00:00
[2021-02-18 22:48:04,362] {logging_mixin.py:103} INFO - Autenticação no twitter
[2021-02-18 22:48:04,377] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:48:04,564] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:48:05,113] {logging_mixin.py:103} INFO - 200
[2021-02-18 22:48:05,124] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:48:05,280] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:48:05,637] {logging_mixin.py:103} INFO - 400
[2021-02-18 22:48:05,645] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:48:05,814] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:48:06,524] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:48:06,527] {logging_mixin.py:103} INFO - 0:00:02.166508
[2021-02-18 22:48:06,528] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:48:08,533] {logging_mixin.py:103} INFO - 0:00:02.005138
[2021-02-18 22:48:08,561] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:48:08,583] {logging_mixin.py:103} INFO - 0:00:00.050203
[2021-02-18 22:48:08,583] {logging_mixin.py:103} INFO - 600 devidamente inseridas no banco
[2021-02-18 22:48:08,585] {python.py:118} INFO - Done. Returned value was: None
[2021-02-18 22:48:08,589] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=tweets_by_user, task_id=get_tweets_by_user_list, execution_date=20210218T224700, start_date=20210218T224804, end_date=20210218T224808
[2021-02-18 22:48:08,610] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-18 22:48:08,650] {local_task_job.py:118} INFO - Task exited with return code 0
